audio.go

```diff
@@ -47,19 +48,44 @@ func (p *soundMgr) init(g *Game) {
 }
 
 func (p *soundMgr) play(media Sound, opts *PlayOptions) (err error) {
-	err = p.playSfx(media, opts.Wait, false)
+	if opts.Music {
+		err = p.playBgm(media, opts.Action)
+	} else {
+		err = p.playSfx(media)
```

Comment:
Is this a breaking change for existed API `Game.Play`?

If `Game.Play` is called with the original way (without specifying field `Music`), that results in different behavior than spx v1.

Reply:
Currently, audio only has a play function, (and pause, resume, etc., have been removed). From a typical game development perspective, the need to pause audio is unnecessary. I've developed many games and never needed it.

If you all consistently believe that this SFX needs a pause feature, I'll add it later.

Reply:
> If you all strongly believe that this SFX requires a pause feature

I'm uncertain about the necessity of a pause feature, actually I haven't used that either. But I don't think it's the right time to implement a breaking change to the APIs now.

> I will add it later.

No problem.